r/vignettes/flight.Rmd

```diff
@@ -74,5 +82,6 @@ client %>%
 ## 3 virginica        6.9
 ```
 
-Because `flight_get()` returns an Arrow data structure, we can directly pipe
-its result into a `dplyr` workflow.
+Because `flight_get()` returns an Arrow data structure, you can directly pipe
+its result into a [dplyr](https://dplyr.tidyverse.org/) workflow.
+See [the dataset vignette](./dataset.html) for more information on working with Arrow objects via a dplyr interface.
```

Comment:
pkgdown will linkify this correctly and it should work around any quirks in how CRAN makes URLs

```suggestion
See `vignette("dataset", package = "arrow")` for more information on working with Arrow objects via a dplyr interface.
```

Reply:
Thanks!